extension_templates/forecasting.py

```diff
@@ -90,8 +90,10 @@ class MyForecaster(BaseForecaster):
         "requires-fh-in-fit": True,  # is forecasting horizon already required in fit?
         "X-y-must-have-same-index": True,  # can estimator handle different X/y index?
         "enforce_index_type": None,  # index type that needs to be enforced in X/y
+        "capability:pred_int": False,  # does forecaster implement predict_quantiles?
+        # in the future can be changed to capability:predict_quantiles
```

Comment:
bit unclear what this means to the person who wants to implement the forecaster. Perhaps say sth like "deprecated and will be renamed to `capability:predict_quantiles` in 0.11.0"?